projects/bp-gallery/src/components/views/location-curating/LocationManagementDialog.tsx

```diff
@@ -0,0 +1,392 @@
+                      <TextField
+                        className='location-management-synonyms-input'
+                        variant='standard'
+                        margin='dense'
+                        onKeyDown={(event: any) => {
+                          if (event.key === 'Enter' && event.target.value.length > 0) {
+                            addSynonym(event.target.value as string);
+                            event.target.value = '';
+                          }
+                        }}
+                      />
+                    </div>
+                  </PictureInfoField>
+                </div>
+                <div className='location-management-children-container'>
+                  <div>{t('common.sublocations')}</div>
+                  <PictureInfoField
+                    title={t('common.sublocations')}
+                    icon={<Place />}
+                    type='location'
+                  >
+                    <TagSelectionField
+                      type={TagType.LOCATION}
+                      tags={
+                        (locationTag.child_tags?.map(tag => ({ ...tag, verified: true })) as any) ??
+                        []
+                      }
+                      allTags={(flattenedTags as any) ?? []}
+                      onChange={locations => {
+                        setChildTags(locations as FlatTag[]);
+                      }}
+                      noContentText={''}
+                      fixedParentTag={locationTag}
+                      customChipOnClick={(id: string) => {
+                        parentTagHistory.current.push(parentTag);
+                        setParentTag(locationTag);
+                        setLocationTagID(id);
+                      }}
+                      allowCreateChild={true}
+                    />
+                  </PictureInfoField>
+                </div>
+                <div className='location-management-parents-container'>
+                  <div>{t('common.superlocations')}</div>
+                  <PictureInfoField
+                    title={t('common.superlocations')}
+                    icon={<Place />}
+                    type='location'
+                  >
+                    <TagSelectionField
+                      type={TagType.LOCATION}
+                      tags={
+                        (locationTag.parent_tags?.map(tag => ({
+                          ...tag,
+                          verified: true,
+                        })) as any) ?? []
+                      }
+                      allTags={(flattenedTags as any) ?? []}
+                      onChange={locations => {
+                        setParentTags(locations as FlatTag[]);
+                      }}
+                      noContentText={''}
+                      fixedChildTag={locationTag}
+                      customChipOnClick={(id: string) => {
+                        setParentTag(parentTagHistory.current.pop());
```

Comment:
Yeah. I have already noticed that problem, when writing the code. I thought that providing some guidance in most cases would still be helpful, especially because I believe, that the displayed case will be less relevant on our current dataset, even though it will pop up for sure. I also think, that just displaying all siblings will be more harmful, because this would allow getting lost and potentially increases the risk of unintentionally adding, deleting, or modifying location tags somewhere in the data structure. Maybe I can come up with a better solution, but I would not feel unhappy with sticking with this solution.